setup.py

```diff
@@ -55,9 +55,23 @@ def get_version():
         for line in lines:
             if line.startswith('Version:'):
                 return line[8:].strip()
-    if os.path.exists("./tools"):
-        print("Running tools/get_version.sh")
-        version = subprocess.getoutput("bash -c 'cd tools; ./get_version.sh'")
+    with open("./include/vrv/vrvdef.h") as header_file:
+        defines = {}
+        for line in header_file:
+            if not line.startswith("#define"):
+                continue
+            definition = line.strip().split()
+            if len(definition) > 2:
```

Comment:
The 'else' is redundant; you can `continue` if you **don't** meet the requirements, then the code will stop processing there.

```
if len(segments) < 3:  # if it's fewer than three, something went wrong.
    continue

defines[definition[1]] = definition[2]
```